base/pkg/dir.jl

```diff
@@ -54,8 +59,14 @@ function init(meta::AbstractString=DEFAULT_META, branch::AbstractString=META_BRA
                 close(io)
             end
         end
+        #Move TEMP to METADATA
+        Base.mv(joinpath(temp_dir,"METADATA"), metadata_dir)
+        Base.mv(joinpath(temp_dir,"REQUIRE"), joinpath(metadata_dir,"REQUIRE"))
+        Base.mv(joinpath(temp_dir,"META_BRANCH"), joinpath(metadata_dir,"META_BRANCH"))
```

Comment:
I think `REQUIRE` and `META_BRANCH` should be moved to `joinpath(dir, ...)` instead of `metadata_dir` ?

@StefanKarpinski can you take a look at this?